blazor/scheduler/appointments.md

```diff
@@ -477,6 +477,63 @@ By default, the scheduler will render the overlapping events based on the start
 ```
 ![Blazor Scheduler with Overlapping Events](images/blazor-scheduler-overlapping-events.png)
 
+## Preventing Overlapping Events
+
+By default, the scheduler renders overlapping events based on their start and end times. To prevent this, set the [AllowOverlap](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Schedule.SfSchedule-1.html#Syncfusion_Blazor_Schedule_SfSchedule_1_AllowOverlap) property to `false`. If new or updated events conflict with existing ones, an overlap alert will be displayed when this property set as `false`. The conflicting events will be collected in the [OverlapCollection](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Schedule.PopupOpenEventArgs-1.html#Syncfusion_Blazor_Schedule_PopupOpenEventArgs_1_OverlapCollection) within the [PopupOpenEventArgs](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Schedule.PopupOpenEventArgs-1.html).
+
```

Comment:
By default, the scheduler displays overlapping events according to their start and end times. To avoid this behavior, you can set the AllowOverlap property to false. When this property is set to false, an overlap alert will appear if new or updated events conflict with existing ones. The conflicting events will be gathered in the OverlapCollection inside the PopupOpenEventArgs.